lib/node-utils.ts

```diff
@@ -253,3 +254,42 @@ export function getImportModuleName(
     return node.arguments[0].value;
   }
 }
+
+type AssertNodeInfo = {
+  matcher: string | null;
+  isNegated: boolean;
+};
+/**
+ * Extracts matcher info from MemberExpression node representing an assert.
+ */
+export function getAssertNodeInfo(
+  node: TSESTree.MemberExpression
+): AssertNodeInfo {
+  const emptyInfo = { matcher: null, isNegated: false } as AssertNodeInfo;
+
+  if (
+    !isCallExpression(node.object) ||
+    !ASTUtils.isIdentifier(node.object.callee)
+  ) {
+    return emptyInfo;
+  }
+
+  if (node.object.callee.name !== 'expect') {
+    return emptyInfo;
+  }
+
+  let matcher = ASTUtils.getPropertyName(node);
+  let isNegated = false;
+  if (matcher === 'not') {
```

Comment:
Nit, I do find this more readable because it isn't reassigned


```suggestion
  let isNegated = matcher === 'not';
  if (isNegated ) {
```

Reply:
That's better, I'm changing it!